crates/edr_rpc_server/src/server.rs

```diff
@@ -0,0 +1,696 @@
+use std::net::{SocketAddr, TcpListener};
+use std::sync::Arc;
```

Comment:
Merge imports

Reply:
Done in [Merge imports](https://github.com/NomicFoundation/hardhat/pull/4539/commits/0e0a2f349ef63b628abbab1547476d3a621afd6d)